torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -457,13 +616,16 @@ def identify_mutated_tensors(kernel, kwargs):
         import traceback
 
         warnings.warn(
-            "Encountered an exception in identify_mutated_tensors, assuming every input is mutated"
-        )
-        log.debug(
+            "Encountered an exception in identify_mutated_tensors, "
+            "assuming every input is mutated:\n"
             "".join(
                 traceback.TracebackException.from_exception(e).format()  # noqa: G001
             )
         )
+        if ttir_module is not None:
+            log.debug("TTIR: %s", str(ttir_module))
```

Comment:
```suggestion
            log.debug(f"TTIR: \n%s", str(ttir_module))
```

Reply:
Linter complains when trying to use f-strings in log messages.